src/util.h

```diff
@@ -48,6 +48,13 @@ extern bool fLogIPs;
 extern volatile bool fReopenDebugLog;
 extern CTranslationInterface translationInterface;
 
+static const char BITCOIN_CONF_FILENAME[] = "bitcoin.conf";
```

Comment:
Declaring the constant and its value in a header won't work well for data structures that require a larger amount of memory or indirected pointers, such as strings, structs, objects etc. In worst case this causes errors in some compilers, in best case this causes unnecessary memory use.